src/app/[locale]/_hooks/__tests__/usePushBack.test.tsx

```diff
@@ -0,0 +1,14 @@
+import { render, renderHook } from '@testing-library/react';
+import usePushBack from '../usePushBack';
+
+jest.mock('next-intl/client', () => ({
+    useRouter() {
+      return {
+        back: () => jest.fn(),
+      };
+    },
+  }));
+
+test('render hook', () => {

```

Comment:
What do you think to add a description and more meaningful texts? This'd help future us reading these tests again ;)